core/genesis.go

```diff
@@ -176,8 +176,9 @@ func SetupGenesisBlock(db ethdb.Database, genesis *Genesis) (*params.ChainConfig
 		}
 	}
 
-	// Get the existing chain configuration.
+	// Get the new chain configuration for genesis block.
 	newcfg := genesis.configOrDefault(stored)
+	// Get the existing chain configuration for genesis block.
```

Comment:
The original comment was fine, you don't need to explicitly say what every line does. The goal is to get the chain config, the following lines do it. Fine to leave as is.

Reply:
OK. Accept your comment. The original comment is fine. I will revert this modification back and re-commit again, because there are still other two spelling mistakes modification.

Reply:
I have already revert this comment modification back. And re-commit the other two spelling mistakes. There are four commit because my misoperation in my local PC by using "git commit --amend". You can just ignore it. There are only two small spelling modification though.